spec/rubydoop/job_definition_spec.rb

```diff
@@ -77,6 +77,34 @@ module Rubydoop
         job_definition.output('path', format: :sequence_file)
         configuration.get('mapreduce.outputformat.class').should == 'org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat'
       end
+
+      it 'should raise ArgumentError if only given options' do
+        expect { job_definition.output(format: :text) }.to raise_error(ArgumentError)
+      end
+
+      context 'with intermediate paths' do
+        it 'adds a unique suffix to the path' do
+          job_definition.output('path', intermediate: true)
+          job_definition.output.should =~ /\Apath-\d{10}-\d{5}\Z/
+        end
+
+        it 'should default to the job name when dir is not set' do
```

Comment:
It's hard for me to say I don't agree with what you're saying, and I will consistently use assertive descriptions throughout.